fastlane/jetpack_metadata/ja/release_notes.txt

```diff
@@ -0,0 +1,11 @@
+ブロックエディターに小さいながらも重要な変更を加えました。ブロックを選択していない状態でエディターを初めて開いたときに、「ブロックを追加」ボタンがこれまでよりもわかりやすく表示されます。 また、メディアブロックや音声ブロックをアップロードできなかったときに表示される3つのエラー通知のうち、不要と思われるものを1つ削除しました。
+
+VoiceOver を使ってメニュー項目を並べ替える際に、アクセシビリティの調整に関するガイダンスが再生されます。 手順や注意点が明確になり、メニュー階層を音声で確認しやすくなりました。
+
+サイト作成時に、そのサイトの目的を入力できる画面を新たに追加しました。 この画面は小規模のユーザーグループで検証されたものですが、皆さまにも気に入っていただけると思います。
+
+ブラウザーのツールバーが表示されていると、ウェブプレビューで画面下部の通知が見えなくなってしまう問題が解消されました。
+
+名前にはどんな意味が込められていますか ? サイト名に由来がある場合、「参加サイト」のナビゲーションタイトルに表示できます。
+
+通知のコメントを左にスワイプしたときの「ゴミ箱へ移動」オプションが表示されなくなりました。 今後は「コメントを承認しない」と「コメントを承認する」が代わりに表示されます。 当社で承認を行います。
```

Comment:
For the record, this and all the other localized release notes apart from `pt-BR` were copied from the Jetpack version via `ed648765ba` because not yet available in GlotPress at the time of writing.

This is actually a known thing with the GlotPress tooling and having the same set of release notes for WordPress and Jetpack. See p8Qyks-2BD-p2#comment-1006